pkg/config/templates/rancherd-13-monitoring.yaml

```diff
@@ -39,6 +39,13 @@ resources:
           type: pvc
           accessModes:
           - ReadWriteOnce
+        resources:
+          limits:
+            cpu: 200m
+            memory: 500Mi
```

Comment:
just curious, how we know 500Mi is the suitable value for grafana?

Reply:
yap, I feel it to be a bit higher :p 

And it's also good because we could modify it via `kubectl` after this commit.
Thanks for your explanation.